### PR TITLE
Added installation instructions and clarified usage

### PR DESCRIPTION
**Pull request by geekygirldawn**
I added just a few lines to the README with installation instructions and a mention that maintainers looks for OWNERS files in the current directory when running it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?
 
 and print the github id, pr comment count and devstats contribution count as well.
 
+## Installation
+
+Run `make` to install Maintainers. Currently `make` will run the `hack/install-maintainers.sh` script.
```
Comment (MadhavJivrajani) [inline README.md]:
```suggestion
Run `make` to install `maintainers`. Currently `make` will run the `hack/install-maintainers.sh` script.
```
nit

File: README.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?
...
+
+## Usage
+
+Maintainers looks for OWNERS files in the repository where you run `maintainers`. In the
```
Comment (MadhavJivrajani) [inline README.md]: There are commands like `check-url` which don't need to interact with OWNERS files. Maybe we drop the Usage section or we could word it differently to be generic enough?

**Review by MadhavJivrajani — COMMENTED** [review]
Thank you! ❤️

File: README.md (hunk 1 above)
Comment (geekygirldawn) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (geekygirldawn) [reply README.md]: I've made this more generic and not specific to OWNERS files. How does this new version look?

File: README.md [hunk 3]
```diff
@@ -5,6 +5,16 @@ https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?
...
 and print the github id, pr comment count and devstats contribution count as well.
 
+## Installation
+
+Run `make` to install `maintainers`. Currently `make` will run the `hack/install-maintainers.sh` script.
+
+## Usage
+
+Unless there is a flag to specify the context (e.g., --repository-github, --yaml-file,
+--kubernetes-directory), Maintainers uses the directory where you run `maintainers` to provide the context
```
Comment (MadhavJivrajani) [inline README.md]:
```suggestion
--kubernetes-directory), the tool uses the directory in which it is run to provide context
```

**Review by MadhavJivrajani — COMMENTED** [review]
Small nit, LGTM otherwise! Thanks again!

File: README.md (hunk 3 above)
Comment (geekygirldawn) [reply README.md]: I made this small change.

**Comment by MadhavJivrajani** [comment]
/label tide/merge-method-squash 
/lgtm
/approve
Sorry for the delay, and thanks again! :heart:

**Pull request merged**